libs/backend/src/ui_strings/ui_strings_machine_configuration.ts

```diff
@@ -0,0 +1,41 @@
+/* istanbul ignore file - tested via VxSuite apps. */
+
+import { BallotPackage, LanguageCode } from '@votingworks/types';
+import { Logger } from '@votingworks/logging';
+import { UiStringsStore } from './ui_strings_store';
+
+/** Input for {@link processBallotPackage}. */
```

Comment:
I can't seem to find a function with the name `processBallotPackage`, so I'm wondering if this `{@link processBallotPackage}` is a stale reference - maybe you're planning to add this function in the future?

Reply:
Thanks! You're right about the stale reference - will update